src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -24,6 +24,16 @@
 ## Version 4.4.1
 * Set minimum dependency of module to PowerShell 5.0
 
+## Version 4.5.0
+* Update all Azure Sql Database/ElasticPool Creation/Update related cmdlets to use the new Database API, which support Sku property for scale and tier-related properties.
```

Comment:
@payiAzure this snippet you are adding should go underneath the `## Current Release` header

Reply:
@cormacpayne will put this under the ## Current Release